File: dotNET/PFProSDK/src/PayPal/Payments/Communication/SendReconnectState.cs
#region "Imports"

#endregion

namespace PayPal.Payments.Communication
{
	/// <summary>
	/// Represents SendReconnectState
	/// </summary>
	internal class SendReconnectState : ReconnectState
	{
		#region "Construcotrs"

		/// <summary>
		/// Copy Constructor for SendInitState
		/// </summary>
		/// <param name="CurrentPaymentState">Current Payment State object.</param>
		public SendReconnectState(PaymentState CurrentPaymentState) : base(CurrentPaymentState)
		{
		}

		#endregion
	}
}

File: java/src/sdk/base/paypal/payflow/SendReconnectState.java
package paypal.payflow;



/**
 * Represents SendReconnectState
 */
class SendReconnectState extends ReconnectState {

    /**
     * Copy Constructor for SendInitState
     *
     * @param currentPaymentState PaymentState
     */
    public SendReconnectState(PaymentState currentPaymentState) {
        super(currentPaymentState);
        Logger.getInstance().log("paypal.payflow.SendReconnectState.SendReconnectState(PaymentState) : Entered", PayflowConstants.SEVERITY_DEBUG);
        Logger.getInstance().log("paypal.payflow.SendReconnectState.SendReconnectState(PaymentState) : Exiting", PayflowConstants.SEVERITY_DEBUG);
    }

}
